lms/djangoapps/teams/static/teams/js/views/teams_tab.js

```diff
@@ -71,6 +73,7 @@
                         ['topics/:topic_id/search(/)', _.bind(this.searchTeams, this)],
                         ['topics/:topic_id/create-team(/)', _.bind(this.newTeam, this)],
                         ['topics/:topic_id/:team_id/edit-team(/)', _.bind(this.editTeam, this)],
+                        ['topics/:topic_id/:team_id/edit-team/manage-members(/)', _.bind(this.editTeamMembers, this)],
```

Comment:
I'm ambivalent on leaving these out of the router for non-privileged users. One the one hand, the UI is somewhat confusing when trying to edit a team without privileges, but on the other, it won't be very often that they will manage to see the edit team UI. We also will probably end up making the UI respond more coherently to server errors in [TNL-3042](https://openedx.atlassian.net/browse/TNL-3042).